### ISSUE TITLE
Thank you!!!

### ISSUE THREAD

**Issue by ped998** [bseltz-cohesity/scripts#98]
Not an "issue", but wanted to let you know that upgradeAgents.py saved me a **ton** of time analyzing the certificate expiry issue.

Thank you!!!

**Comment by bseltz-cohesity** [comment]
Good to hear! Also check out https://github.com/bseltz-cohesity/scripts/tree/master/python/agentCertificateCheck

**Comment by bseltz-cohesity** [comment]
Hey @ped998 can you tell me how you used the script? Did you just run it in report mode and review the CSV output? Did you use any of the other parameters / filters, or execute any upgrades with it? Thanks for the feedback!

**Comment by ped998** [comment]
I used the the script to determine which agents were impacted by the Certificate issue.  Called the script iteratively as I looped through our list of about 100 clusters, and then grep'd out the "not impacted" lines and concatenated results into a single csv to pass on to the Compute & Storage teams.   I know there is the Helios option for the scripts that could have done all clusters in a single pass, however we use SSO, and I have not gotten them to work with that.   

Basically, within about 30 minutes of reading the support note (containing the reference to the scripts), I was able to have a list of impacted agents, and I knew that 80% of our agents did not have this issue.
